pelican/generators.py

```diff
@@ -680,21 +681,9 @@ class StaticGenerator(Generator):
 
     def __init__(self, *args, **kwargs):
         super(StaticGenerator, self).__init__(*args, **kwargs)
+        self.create_symlinks = False
```

Comment:
Well, we **can always** create symlinks, it's just that sometimes we **must**. `fallback_to_symbolic_links`?